site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,157 @@ updated: 2021-08-04
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
-### Review violation troubleshooting updates
-
-#### Published 2021.10.27
+### Chrome 96: Dynamic content scripts {: #m96-dynamic-content-scripts }
+
+The [`chrome.scripting`](/docs/extensions/reference/scripting/) API now supports
+[registering](/docs/extensions/reference/scripting/#method-registerContentScripts),
+[updating](/docs/extensions/reference/scripting/#method-updateContentScripts),
+[unregistering](/docs/extensions/reference/scripting/#method-unregisterContentScripts), and [getting
+a list of content scripts](/docs/extensions/reference/scripting/#method-getRegisteredContentScripts)
+at runtime. Previously, content scripts could only be statically declared in an extension's
+manifest.json or programmatically injected at runtime with
+[`chrome.scripting.executeScript`](/docs/extensions/reference/scripting/#method-executeScript).
+
+### Chrome 96: Expanded promise support {: #m96-promise-support }
+
+A number of general and Chrome OS-specific extensions APIs have gained promise support in this
+release. Expand the following sections for more details.
+
+{% Details %}
+{% DetailsSummary %}
+Extensions APIs
+{% endDetailsSummary %}
+
+A number of APIs now have promise support in Manifest V3.
```

Comment:
Are these classes, methods, or properties? A heuristic I use is to only use 'API' for references to multiple item types. (Language should always be specific.) 

If any of these are methods they should have '()' after them.

Reply:
These are all "namespaces", but we generally refer to them as "APIs" - e.g. the tabs API, the history API, etc.  This is consistent with our language elsewhere.

Reply:
'Namespaces' is something I never would have guessed. A writing best practice has always been to prefer the more specific term. In an electronic world, a more specific term results in fewer search results. It's an aid to finding information.

Reply:
"APIs" is the canonical term for these.